client/src/ui-components/CreateProductForm/CreateProductForm.css

```diff
@@ -0,0 +1,8 @@
+.createProductForm {
+  text-align: center;
+  width: 400px;
+  margin-top: 50px;
+}
+.title {
+  text-align: center;
+}
```

Comment:
change the name of this class, like title-product-form, because title is already used somewhere else